System/GraphicalPrimitive.cpp

```diff
@@ -31,13 +31,43 @@ namespace RTE {
 		drawRightPos.m_Y -= targetPos.m_Y;
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	static void lineWithThickness(BITMAP *drawScreen, const Vector &startPos, const Vector &endPos, unsigned char color, const int &thickness) {
```

Comment:
Rather than a static function, I'd rather a static private method (if cpp allows that).
Also, there's no reason for thickness to be a `const int &`, it's not being modified in the function and there's no need to pass an int by reference instead of copy.

Reply:
Static private member would discard internal linkage and won't be much of a use if it is private (means we woudln't be able to call it statically ouside of `LinePrimitive` class) + I was thinking maybe some functionality could benefit from *publicly* defined static member function but couldn't think of any, so limited this function as helper in scope of a single translation unit.

Reply:
If anything ever needs this concept in future, we should find a better place to put it, e.g. RTETools. GraphicalPrimitive should be concerned solely with stuff related to graphical primitives, it's job isn't to provide a globally accessible line function.

I'm also fine with this just being a private method, I suggested static since it's already static and doesn't need any object-scope stuff, but you're right that there's not a real benefit to private static here over private non-static.